### PR TITLE
Create ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58410](https://cumulocity.atlassian.net/browse/MTM-58410)
Original PR: [6331](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6331)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Data points graph realtime disabled when tab is in background (#6288) [GRAFT][release/y2024] (#6331)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Data points graph realtime updates disabled when browser tab is in background
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58410
+version: 1018.503.88
+---
+Data points graph realtime disabled when tab is in background (#6288) [GRAFT][release/y2024] (#6331)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve performance and reduce unnecessary resource consumption, the data points graph will no longer update in realtime when the browser tab containing the application is in the background or not visible to the user. The graph will resume realtime updates once the tab is brought back into focus. This change optimizes system resources by avoiding unnecessary graph rendering when the user is not actively viewing the application tab, resulting in improved overall performance and responsiveness of the application.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md (hunk 1 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:3]:
```suggestion
title: Data points graph realtime is disabled when tab is in background issue fix.
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md (hunk 2 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]: Data points graph realtime was automatically disabled when tab was in the background. With this fix, realtime is kept enabled and can be disabled only by direct user's interaction.

Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]:
```suggestion
Data points graph realtime was automatically disabled when tab was in the background. With this fix, realtime is kept enabled and can be disabled only by direct user's interaction.
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58410
+version: 1018.503.88
+---
+Data points graph realtime was automatically disabled when tab was in the background. With this fix, realtime is kept enabled and can be disabled only by direct user's interaction.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]:
```suggestion
The realtime functionality for the data points graph was automatically disabled when the tab was in the background. With this fix, realtime is kept enabled and can be disabled only by direct user interaction.
```

Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]: What is the context, what does "the tab" mean here? Is it in the Data points graph widget? On a dashboard?

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Data points graph realtime is disabled when tab is in background issue fix.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:3]:
```suggestion
title: Data points graph realtime is kept enabled when the tab is in background
```

Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:3]:
```suggestion
title: Data points graph realtime is kept enabled when the browser tab is in background
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58410
+version: 1018.503.88
+---
+Data points graph realtime was automatically disabled when tab was in the background. With this fix, realtime is kept enabled and can be disabled only by direct user's interaction.
```
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]:
```suggestion
The realtime functionality for the data points graph was automatically disabled when the browser tab was in the background. With this fix, realtime is kept enabled and can be disabled only by direct user interaction.
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md (hunk 3 above)
Comment (jdreSAG) [reply content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md]: @BeateRixen it's browser tab. I commited your version with 'browser' word added

File: content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md (hunk 4 above)
Comment (jdreSAG) [reply content/change-logs/application-enablement/ui-c8y-1018-503-88-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md]: @BeateRixen I commited your version with 'browser' word added

**Review by sag-lukjan — APPROVED** [review]

**Pull request merged**